deploy/prod/cla-assistant.yaml

```diff
@@ -0,0 +1,74 @@
+replicaCount: 1
+service:
+  enabled: true
+  name: cla-assistent
+  port: 8080
+  type: ClusterIP
+  containerPort: 8080
+  tls:
+    enabled: true
+    issuerName: cloudflare
+image:
+  repository: ${ECR_REGISTRY}/cla-assistent-prod
```

Comment:
prod should be a tag

Reply:
@eskp sorry, didn't get it. `prod` is set as tag now, have a look on line `14`

Reply:
This is related to changing the ECR name and getting rid of "prod" in the name